keyboards/willoucom/keypad/info.json

```diff
@@ -0,0 +1,70 @@
+{
+    "manufacturer": "Willou.com",
+    "keyboard_name": "keypad",
+    "maintainer": "willoucom",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP0", "GP1", "GP2", "GP3", "GP4"],
+            ["GP5", "GP6", "GP7", "GP8", "GP9"],
+            ["GP10", "GP11", "GP12", "GP13", "GP14"],
+            ["GP15", "GP16", "GP17", "GP18", "GP19"],
+            ["GP20"],
+            ["GP21"],
+            ["GP22", "GP26"],
+            ["GP27"],
+            ["GP28"]
+        ]
+    },
+    "processor": "RP2040",
+    "tapping": {
+        "hold_on_other_key_press": true
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "force_nkro": true,
+        "pid": "0x4B50",
+        "vid": "0x574A"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
```

Comment:
Key index is out of order.
![image](https://github.com/qmk/qmk_firmware/assets/4781841/0a511481-db90-4939-b383-2e90e9a09a7b)

Reply:
![IMG_20230814_122623](https://github.com/qmk/qmk_firmware/assets/4134696/79dd6d1d-3293-4de7-b8a7-2c03ed5e8e71)

For keys 20-28, on the product they are positioned vertically on a daughterboard which can be moved and rotated. To make the visual more attractive I made the decision to use the horizontal space and move this section up to make the configurator easier to use.
Should I position this section below the pad grouping the keys 0 to 19?

Reply:
The issue isn't the physical positioning.

Reply:
I guess it's the labels that are wrong, I fixed these as follows
![image](https://github.com/qmk/qmk_firmware/assets/4134696/e5abaf9e-7076-486d-aafb-8bef49ffe45e)

Reply:
Please can you explain, I don't understand what you mean.

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/3e8fda36-294e-427f-a827-08111bce6f1f)